hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -114,11 +115,18 @@ public SignedStateBalancesExporter(
 	@Override
 	public boolean isTimeToExport(Instant now) {
 		if (periodBegin != NEVER
-				&& now.getEpochSecond() / exportPeriod != periodBegin.getEpochSecond() / exportPeriod) {
+				&& now.getEpochSecond() / exportPeriod != periodBegin.getEpochSecond() / exportPeriod
+		        && now.getEpochSecond() % exportPeriod <= SCHEDULED_EXPORT_ALLOWED_SKEW) {
+			log.info(String.format("Now %s is time to export. periodBegin = %s, exportPeriod = %d.",
+					now.toString(), periodBegin.toString(), exportPeriod));
+
```

Comment:
This change, which is on the server side, should be unit tested.

Reply:
Will add unit test cases for the check.